### ISSUE TITLE
is the plugin supposed to work in offline installations?

### ISSUE THREAD

**Issue by johnqa** [reportportal/plugin-bts-azure#30]
Hi,

I compiled and installed the plugin in an offline deployment of Report Portal.

When I try to add an integration I can only see an empty page with some errors in dev console pointing to reactjs.org

Does that mean the plugin only works if it has access to internet?

Thank you,
John

**Comment by samagarw** [comment]
Yes plugin build is successful but I am getting a blank page on adding integration . 

Can anyone please help here .

**Comment by Nils1970** [comment]
I am struggling with setup of on-premise integration with Azure DevOps Server 2020. It looks like the APIClient has a hardcover BaseUrl set to dev.azure.com .There is a public setter to set BaseUrl, but configuration dialog  in Reportportal does not allow to set BaseUrl. really  a pity, because  my experience is that rest API works the same for Azure DevOps Services and on-premise Azure DevOps Server (which can have of course a different BaseUrl)
